src/NHibernate/Dialect/Dialect.cs

```diff
@@ -2480,6 +2480,14 @@ public virtual string AddColumnString
 			get { throw new NotSupportedException("No add column syntax supported by Dialect"); }
 		}
 
+		/// <summary>
+		/// The syntax for the suffix used to add a column to a table. Note this is deprecated
```

Comment:
The "Note" copied from `AddColumnString` own comment seems to be a note for some deprecation which does not seem to be done NHibernate side. Better remove that note in this new property comment.

Reply:
I just added the deprecation note because it was already present on `AddColumnString` and `AddColumnSuffixString` doesn't make sense without `AddColumnString`. Do you want me to remove the deprecation note on both properties? Having the deprecation note on only one property seems wrong to me.

Reply:
A correct deprecation note is the `Obsolete` attribute anyway. Ok for removing both.